CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+# Contributing
+
+First of all, thank you for contributing. Any contribution is highly appreciated and welcome.
+
+## Philosophy
+
+There's a simple philosophy behind the Spaceship:
+
+* **Show only what's needed.** The prompt should not be overloaded, it shows only what user needs at the moment (current directory, git branch, etc).
+* **Work out of the box.** The prompt should work right after installation without any additional configuration. Install it and use it.
+* **Be configurable.** It works without configuration, but if a configuration is needed, it provides an easy-to-use interface for customization.
+
+Please, keep this simple rules in mind while you're contributing to Spaceship.
+
+## Setup
+
+1. **Fork** this repo (click the _fork_ button)
+2. **Clone** your fork to your working machine (via `git clone`)
+3. **Add and commit** your contributions
```

Comment:
Suggesting new branch for changes would be nice.

Reply:
Done.